### PR TITLE
PA-2207 Renamed Appriasal Note Label

### PR DESCRIPTION
**Pull request by Fosol**
Renamed label and fixed formatting.

image

### CODE REVIEW

File: frontend/src/features/projects/common/forms/AppraisalCheckListForm.tsx [hunk 1]
```diff
@@ -22,26 +21,21 @@ const AppraisalCheckListForm: React.FunctionComponent<IAppraisalCheckListFormPro
     statusCode: props.taskStatusCode ?? ReviewWorkflowStatus.AppraisalReview,
   });
   return (
-    <Container fluid className={classNames(props.className)}>
```
Comment (Fosol) [inline frontend/src/features/projects/common/forms/AppraisalCheckListForm.tsx:25]: Causing formatting inconsistencies

File: frontend/src/features/projects/common/forms/FirstNationsCheckListForm.tsx [hunk 2]
```diff
@@ -20,17 +19,19 @@ const FirstNationsCheckListForm: React.FunctionComponent<IFirstNationsCheckListF
     statusCode: ReviewWorkflowStatus.FirstNationConsultation,
   });
   return (
-    <Container fluid className={classNames(props.className)}>
```
Comment (Fosol) [inline frontend/src/features/projects/common/forms/FirstNationsCheckListForm.tsx:23]: Causing formatting inconsistencies

**Review by agalbrai — APPROVED** [review]

**Review by goemen — APPROVED** [review]
Looks good to me

File: frontend/src/features/projects/common/forms/AppraisalCheckListForm.tsx (hunk 1 above)
Comment (devinleighsmith) [reply frontend/src/features/projects/common/forms/AppraisalCheckListForm.tsx:25]: Have you double checked that no parent is using this class name prop? If you wanted to keep this more equivalent but remove the container you could add the classname to the row.

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**